SayLess/database.py

```diff
@@ -55,4 +55,9 @@ class Message(db.Model):
     sender = db.Column(db.String(200))
     # rooms should be unique but leaving as such for testing logic
     message = db.Column(db.UnicodeText())
-    mysql_charset='utf8mb4'
\ No newline at end of file
+    mysql_charset='utf8mb4'
+
+class Profile(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    email = db.Column(db.String(200), unique=True)
+    filename = db.Column(db.String(200), unique=True)
```

Comment:
having filename as unique is going to give your issues. You should probably not have this as unique.

Reply:
actually,  this makes sense. You'll probably have to create a way to generate unique filenames in __init__.py tho. Because the user shouldn't have to generate unique filenames

Reply:
Was this resolved? because this will break deployment.